src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -1,5 +1,11 @@
 package games.strategy.engine.framework.startup.mc;
 
+import static games.strategy.engine.framework.ArgParser.CliProperties.TRIPLEA_NAME;
+import static games.strategy.engine.framework.ArgParser.CliProperties.TRIPLEA_PORT;
+import static games.strategy.engine.framework.ArgParser.CliProperties.SERVER_PASSWORD;
+import static games.strategy.engine.framework.ArgParser.CliProperties.TRIPLEA_SERVER;
+import static games.strategy.engine.framework.ArgParser.CliProperties.TRIPLEA_STARTED;
```

Comment:
Please reorder the imports in this file.  Checkstyle is reporting they are not in lexicographical order.